### PR TITLE
[zh] Fix link for /docs/reference/generated

### PR DESCRIPTION
**Pull request by Sea-n**
There are no any translated files under `content/zh-cn/docs/reference/generated/`. It cause 404 not found for those links.

This PR also fixed some obvious format errors.

### CODE REVIEW

File: content/zh-cn/docs/reference/access-authn-authz/kubelet-tls-bootstrapping.md [hunk 1]
```diff
@@ -601,7 +601,7 @@ builtin approver doesn't explicitly deny CSRs. It only ignores unauthorized
 requests. The controller also prunes expired certificates as part of garbage
 collection.
 -->
-作为 [kube-controller-manager](/zh/docs/reference/generated/kube-controller-manager/)
+作为 [kube-controller-manager](/docs/reference/generated/kube-controller-manager/)
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/kubelet-tls-bootstrapping.md:604]: 英文中有重定向，中文没有。

```suggestion
作为 [kube-controller-manager](/zh-cn/docs/reference/command-line-tools-reference/kube-controller-manager/)
```

Comment (Sea-n) [reply content/zh-cn/docs/reference/access-authn-authz/kubelet-tls-bootstrapping.md:604]: Oh, thanks for reminding.

I will check through `static/_redir` again.

```
/docs/reference/generated/cloud-controller-manager/ -> /docs/reference/command-line-tools-reference/cloud-controller-manager/
/docs/reference/generated/kube-apiserver/ -> /docs/reference/command-line-tools-reference/kube-apiserver/
/docs/reference/generated/kube-controller-manager/ -> /docs/reference/command-line-tools-reference/kube-controller-manager/
/docs/reference/generated/kube-proxy/ -> /docs/reference/command-line-tools-reference/kube-proxy/
/docs/reference/generated/kube-scheduler/ -> /docs/reference/command-line-tools-reference/kube-scheduler/
/docs/reference/generated/kubeadm/ -> /docs/reference/setup-tools/kubeadm/
/docs/reference/generated/kubectl/kubectl-options/ -> /docs/reference/kubectl/kubectl/
/docs/reference/generated/kubectl/kubectl/ -> /docs/reference/generated/kubectl/kubectl-commands/
/docs/reference/generated/kubectl/kubectl/kubectl_* -> /docs/reference/generated/kubectl/kubectl-commands#:splat
/docs/reference/generated/kubelet/ -> /docs/reference/command-line-tools-reference/kubelet/
/docs/reference/generated/kubernetes-api/v1.15/ -> https://v1-15.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/
/docs/reference/generated/kubernetes-api/v1.16/ -> https://v1-16.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/
/docs/reference/generated/kubernetes-api/v1.17/ -> https://v1-17.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/
/docs/reference/generated/kubernetes-api/v1.18/ -> https://v1-18.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/
```

File: content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md [hunk 2]
```diff
@@ -117,7 +117,7 @@ of the StatefulSet's Pods.
 下载上面的例子并保存为文件 `web.yaml`。
 
 
-你需要使用两个终端窗口。 在第一个终端中，使用 [`kubectl get`](/zh/docs/user-guide/kubectl/{{< param "version" >}}/#get)  来查看 StatefulSet 的 Pods 的创建情况。
+你需要使用两个终端窗口。在第一个终端中，使用 [`kubectl get`](/zh/docs/user-guide/kubectl/{{< param "version" >}}/#get) 来查看 StatefulSet 的 Pods 的创建情况。
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md:120]: 这个链接无效。

File: content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md [hunk 3]
```diff
@@ -776,8 +776,8 @@ StatefulSet. There are two valid update strategies, `RollingUpdate` and
 
 
 Kubernetes 1.7 版本的 StatefulSet 控制器支持自动更新。
-更新策略由 StatefulSet API Object 的`spec.updateStrategy` 字段决定。这个特性能够用来更新一个 StatefulSet 中的 Pod 的 container images，resource requests，以及 limits，labels 和 annotations。
-`RollingUpdate`滚动更新是 StatefulSets 默认策略。
+更新策略由 StatefulSet API Object 的 `spec.updateStrategy` 字段决定。这个特性能够用来更新一个 StatefulSet 中的 Pod 的 container images，resource requests，以及 limits，labels 和 annotations。
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md:779]: 适当折行

Comment (Sea-n) [reply content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md:779]: This file have many out of sync, I will make another PR for it.

**Comment by Sea-n** [comment]
Since `content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md` have too many out-of-sync, I have removed it for this PR and made another one.
- #34369

**Pull request merged**